assets/js/modules/analytics/components/settings/SettingsForm.stories.js

```diff
@@ -79,6 +79,94 @@ WithoutModuleAccess.args = {
 	hasAnalyticsAccess: false,
 	hasAnalytics4Access: false,
 };
+WithoutModuleAccess.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( MODULES_ANALYTICS_4 ).setPropertyID( null );
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
+
+export const WithoutAnalyticsAccess = Template.bind( null );
+WithoutAnalyticsAccess.storyName = 'Settings w/o Analytics access';
+WithoutAnalyticsAccess.args = {
+	hasAnalyticsAccess: false,
+	hasAnalytics4Access: true,
+};
+WithoutAnalyticsAccess.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( MODULES_ANALYTICS_4 ).setPropertyID( null );
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
```

Comment:
I think this is unnecessary as well.

Reply:
The `propertyID` should be `falsey` to display the GA4 settings notice. See:
https://github.com/google/site-kit-wp/blob/76d40ccc04af6b049b571a61936b42394743cf4d/assets/js/modules/analytics/components/settings/GA4SettingsControls.js#L163

---------

https://github.com/google/site-kit-wp/blob/76d40ccc04af6b049b571a61936b42394743cf4d/assets/js/modules/analytics/components/settings/GA4SettingsControls.js#L272-L277

Reply:
Well if we have covered the case when GA4 is not setup and the Activate Switch is displayed but disabled, then in these cases, we can assume that GA4 is setup. So instead of showing the Activate Switch, it will show the UseSnippetSwitch with the Profile ID in the dropdown. This I think will be the more prominent use case anyways.